docs/references/apps/Connector.md

```diff
@@ -0,0 +1,156 @@
+# Connector
+
+The Connector app can automatically connect interfaces together. It does this using the query mechanisms inside HELICS to detect all the unconnected interfaces in a cosimulation. Then using given database rules it will establish connections between those interfaces.
+
+It can also run in a two phase mode where in the first mode if federates indicate they have potential interfaces. The connector app will go through the potential interfaces to see if there is a potential connection to be made using the same rules. If a potential interface has a connection available it will send a command to the appropriate interface to go ahead and create those interfaces.
```

Comment:
```suggestion
It can also run in a two phase mode if federates indicate they have potential interfaces. In the first phase, the connector app will go through the potential interfaces to see if there is a potential connection to be made using the same rules. If a potential interface has a connection available it will send a command to the appropriate interface to go ahead and create those interfaces.
```